{{cookiecutter.extension_name}}/RELEASE.md

```diff
@@ -0,0 +1,38 @@
+# Making a {{ cookiecutter.extension_name }} release
+
+This document guides a contributor through creating a release of {{ cookiecutter.extension_name }}.
+
+
+We publish a Python source package and a Python universal binary wheel.
+
+## Update version number
+
+Update the version number in `setup.py` and in `package.json`.
+
+## Remove generated files
+
+Do `npm run clean` to remove any old Javascript builds. Delete the `dist/` folder to remove old python package builds.
+
+```bash
+npm run clean
+rm -rf dist/
+```
+
+## Build the package
+
+First build the Javascript extension bundle, then build the python package and wheel.
```

Comment:
Using the following, build the JavaScript extension bundle and the Python package and wheel.